services/ui-src/src/components/export/ExportedModalDrawerReportSection.tsx

```diff
@@ -17,21 +17,33 @@ export const ExportedModalDrawerReportSection = ({
   const sectionHeading = verbiage?.intro.subsection || name;
   const entityCount = report?.fieldData?.[entityType]?.length;
 
-  let emptyEntityMessage = "";
+  let emptyEntityMessage;
```

Comment:
The default case will leave this undefined. is that intentional?

Reply:
It was a change I made when I was trying to refactor a bit and didn't change it back. I can set it to a string though